strawberry_django/auth/mutations.py

```diff
@@ -28,29 +30,30 @@
 
 
 @django_resolver
-def resolve_login(info: Info, username: str, password: str) -> AbstractBaseUser | None:
+def resolve_login(info: Info, username: str, password: str) -> AbstractBaseUser:
     request = get_request(info)
     user = auth.authenticate(request, username=username, password=password)
 
-    if user is not None:
+    if user is None:
+        raise IncorrectUsernamePasswordError()  # noqa: RSE102
```

Comment:
I don't think we should be creating our own exceptions here, but instead using django's `ValidationError`

If we want custom exceptions (is there a good reason for one?) then we should inherit those from `ValidationError` directly too

Reply:
That's fine, I don't mind either way.  I find custom exceptions extremely useful for catching errors cleanly - but I do agree they don't expose to the frontend - so not as useful as they could be.

I'll revert it to ValidationError